### ISSUE TITLE
[Bug] Toggle icons are missing next to result summaries in Firefox

### ISSUE THREAD

**Issue by antross** [webhintio/hint#2586]
# 🐛 Bug report

## Description

After running a scan, the toggle icons are missing next to text like "axe accessibility check" in the results. These appear when running the extension in Chrome.

## Environment

* __webhint version:__ 0.0.3
* __Browser version:__ Firefox 67
* __Operating system:__ any
* __URL for which webhint failed:__ any

## `webhint` configuration

<details>
<summary>webhint’s configuration</summary>

Categories:
* [x] Accessibility
* [x] Compatibility
* [x] Performance
* [x] Pitfalls
* [x] PWA
* [x] Security

Your target browsers:
* [x] Recommended settings
* [ ] Custom: 

Ignored resources:
* [x] None
* [ ] Different origin
* [ ] Custom: 

</details>

## Debug output

Please include the content of the `Error details` section if an error message was displayed.

<details>
<summary>webhint’s Error details</summary>

```text
<!-- ✍️ Paste the error details here -->


```

</details>

**Comment by antross** [comment]
Fixed by 1f0cbdcd88e5e6e77d36b34ded4f8c69f68e171f
